### PR TITLE
[FEAT] Commands shortcuts

### PR DESCRIPTION
**Pull request by Guikingone**
Hi everyone 👋 

As notified here: https://github.com/doctrine/DoctrineBundle/issues/1072, here's the PR for the commands shortcuts, I need to finish the tests (on commands that don't have ones for now). 

Feel free to give feedback 🙂

### CODE REVIEW

**Comment by greg0ire** [comment]
Please improve your commit message according to [the contributing guide](https://www.doctrine-project.org/contribute/index.html#working-on-topic-branches).

File: Tests/Command/DropDatabaseDoctrineTest.php [hunk 1]
```diff
@@ -42,6 +43,72 @@ public function testExecute() : void
         );
     }
 
+    public function testExecuteWithAlias()
```
Comment (greg0ire) [inline Tests/Command/DropDatabaseDoctrineTest.php]:
```suggestion
    public function testExecuteWithAlias(): void
```

File: Tests/Command/DropDatabaseDoctrineTest.php [hunk 2]
```diff
@@ -42,6 +43,72 @@ public function testExecute() : void
...
+            sprintf(
+                'Dropped database %s for connection named %s',
+                sys_get_temp_dir() . '/' . $dbName,
+                $connectionName
+            ),
+            $commandTester->getDisplay()
+        );
+    }
+
+    public function testExecuteWithExistAliasButNotSupported()
```
Comment (greg0ire) [inline Tests/Command/DropDatabaseDoctrineTest.php]:
```suggestion
    public function testExecuteWithExistAliasButNotSupported(): void
```

Also, not sure what you want to say with that method name or what you want to test… could you please explain?

Comment (Guikingone) [reply Tests/Command/DropDatabaseDoctrineTest.php]: Yes, actually, the sqlite driver does not support the `-e` option, the idea is to check if the `DbalException` is thrown when using an "invalid" driver.

Comment (greg0ire) [reply Tests/Command/DropDatabaseDoctrineTest.php]: Ok so `testItThrowsWhenUsingIfExistsWithAnIncompatibleDriver`

Comment (Guikingone) [reply Tests/Command/DropDatabaseDoctrineTest.php]: Improved, thanks 🙂

File: Tests/Command/DropDatabaseDoctrineTest.php [hunk 3]
```diff
@@ -42,6 +43,72 @@ public function testExecute() : void
         );
     }
 
+    public function testExecuteWithAlias() : void
```
Comment (ostrolucky) [inline Tests/Command/DropDatabaseDoctrineTest.php]: Nasty copy & paste. Let's make testExecute use dataprovider instead

File: Tests/Command/DropDatabaseDoctrineTest.php [hunk 4]
```diff
@@ -42,6 +43,72 @@ public function testExecute() : void
...
+        $command = $application->find('doctrine:database:drop');
+
+        $commandTester = new CommandTester($command);
+
+        static::expectException(DBALException::class);
+        $commandTester->execute(
+            array_merge(['command' => $command->getName(), '-f' => true, '-e' => true])
+        );
+
+        $this->assertContains(
```
Comment (ostrolucky) [inline Tests/Command/DropDatabaseDoctrineTest.php]: Dead code

File: Tests/Command/DropDatabaseDoctrineTest.php [hunk 5]
```diff
@@ -42,6 +43,72 @@ public function testExecute() : void
...
+            sprintf(
+                'Dropped database %s for connection named %s',
+                sys_get_temp_dir() . '/' . $dbName,
+                $connectionName
+            ),
+            $commandTester->getDisplay()
+        );
+    }
+
+    public function testItThrowsWhenUsingIfExistsWithAnIncompatibleDriver() : void
```
Comment (ostrolucky) [inline Tests/Command/DropDatabaseDoctrineTest.php]: After implementing my first suggestion, I suggest to just do here
```php
public function testItThrowsWhenUsingIfExistsWithAnIncompatibleDriver() : void
{
    static::expectException(DBALException::class);
    $this->testExecute(['-f' => true, '-e' => true]);
}

Comment (Guikingone) [reply Tests/Command/DropDatabaseDoctrineTest.php]: Improved 🙂

File: Command/DropDatabaseDoctrineCommand.php [hunk 6]
```diff
@@ -28,10 +28,10 @@ protected function configure()
         $this
             ->setName('doctrine:database:drop')
             ->setDescription('Drops the configured database')
-            ->addOption('shard', null, InputOption::VALUE_REQUIRED, 'The shard connection to use for this command')
-            ->addOption('connection', null, InputOption::VALUE_OPTIONAL, 'The connection to use for this command')
-            ->addOption('if-exists', null, InputOption::VALUE_NONE, 'Don\'t trigger an error, when the database doesn\'t exist')
-            ->addOption('force', null, InputOption::VALUE_NONE, 'Set this parameter to execute this action')
+            ->addOption('shard', 's', InputOption::VALUE_REQUIRED, 'The shard connection to use for this command')
+            ->addOption('connection', 'c', InputOption::VALUE_OPTIONAL, 'The connection to use for this command')
+            ->addOption('if-exists', 'e', InputOption::VALUE_NONE, 'Don\'t trigger an error, when the database doesn\'t exist')
```
Comment (Pierstoval) [inline Command/DropDatabaseDoctrineCommand.php]: Conflicts with `--env|-e`

Comment (Guikingone) [reply Command/DropDatabaseDoctrineCommand.php]: Hum, maybe `i` ? 🤔

**Comment by Guikingone** [comment]
Just pushed a refactor when it comes to `--if-exists`, as suggested by @Pierstoval , the `-e`conflitcs with `--env`, `i` is used now.

File: Command/DropDatabaseDoctrineCommand.php (hunk 6 above)
Comment (Pierstoval) [reply Command/DropDatabaseDoctrineCommand.php]: I wouldn't even shortcut this specific option 😕

Comment (ostrolucky) [reply Command/DropDatabaseDoctrineCommand.php]: Good point!

File: Command/DropDatabaseDoctrineCommand.php [hunk 7]
```diff
@@ -28,10 +28,10 @@ protected function configure()
         $this
             ->setName('doctrine:database:drop')
             ->setDescription('Drops the configured database')
-            ->addOption('shard', null, InputOption::VALUE_REQUIRED, 'The shard connection to use for this command')
-            ->addOption('connection', null, InputOption::VALUE_OPTIONAL, 'The connection to use for this command')
-            ->addOption('if-exists', null, InputOption::VALUE_NONE, 'Don\'t trigger an error, when the database doesn\'t exist')
-            ->addOption('force', null, InputOption::VALUE_NONE, 'Set this parameter to execute this action')
+            ->addOption('shard', 's', InputOption::VALUE_REQUIRED, 'The shard connection to use for this command')
+            ->addOption('connection', 'c', InputOption::VALUE_OPTIONAL, 'The connection to use for this command')
+            ->addOption('if-exists', 'i', InputOption::VALUE_NONE, 'Don\'t trigger an error, when the database doesn\'t exist')
```
Comment (ostrolucky) [inline Command/DropDatabaseDoctrineCommand.php]: Why is it aliased in this command but not previous one

Comment (Guikingone) [reply Command/DropDatabaseDoctrineCommand.php]: Alias removed

File: Command/DropDatabaseDoctrineCommand.php (hunk 6 above)
Comment (Guikingone) [reply Command/DropDatabaseDoctrineCommand.php]: Alias removed 🙂

Comment (Pierstoval) [reply Command/DropDatabaseDoctrineCommand.php]: 👍

File: Tests/Command/DropDatabaseDoctrineTest.php [hunk 8]
```diff
@@ -3,14 +3,19 @@
...
 use Symfony\Component\Console\Application;
 use Symfony\Component\Console\Tester\CommandTester;
 
 class DropDatabaseDoctrineTest extends TestCase
 {
-    public function testExecute() : void
+    /**
+     * @dataProvider provideForceOption
+     */
+    public function testExecute(array $options) : void
```
Comment (greg0ire) [inline Tests/Command/DropDatabaseDoctrineTest.php]: I've never tried this myself, but I think using variadics here could help simplify the data providers

**Pull request merged**